addons/web/static/src/js/libs/bootstrap.js

```diff
@@ -96,6 +96,16 @@ $.fn.scrollspy.Constructor.prototype.refresh = function () {
     }
 };
 
+/* In some cases, we need to keep the first element of navbar selected (e.g 'Table of Content') */
+
```

Comment:
Remove that blank line and use proper style for documentation comment (and max 80 chars per line for comments):
```javascript
/**
 * ...
 */
```